.github/workflows/release.yml

```diff
@@ -30,23 +32,18 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.repository == 'pmndrs/drei' && contains('refs/heads/master,refs/heads/beta,refs/heads/alpha',github.ref) && github.event_name == 'push' }}
     steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.6.0
-        with:
-          access_token: ${{ secrets.GITHUB_TOKEN }}
       - name: Checkout repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Setup node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
-          node-version: '14.x'
+          cache: 'yarn'
       - name: Install deps
         # this runs a build script so there is no dedicated build
         run: yarn install
       - name: 🚀 Release
-        uses: cycjimmy/semantic-release-action@v2
+        uses: cycjimmy/semantic-release-action@v3
         with:
-          semantic_version: 17
```

Comment:
`semantic_version:` is not recommended anymore since v3: https://github.com/cycjimmy/semantic-release-action#semantic_version